sdk/core/Azure.Core/src/Azure.Core.csproj

```diff
@@ -29,6 +28,11 @@
     <PackageReference Include="System.Memory.Data" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
This is a short-term change.  Please leave the original package reference in place and comment out.   Ensure that an issue is open to track moving back to a project reference and is linked in a comment here.